docs/presigned_urls.md

```diff
@@ -0,0 +1,185 @@
+    :type object_path: string
+    :param expiration: The number of seconds the presigned URL is valid for.
+    :type expiration: int
+    :return: The presigned URL. #TODO: change this
+    :rtype: string
+    """
+
+    # Create low-level S3 service client
+    s3_client = boto3.client("s3")
+
+    try:
+        # URL generated by AWS user with access to private object (i.e. the owner)
+        url = s3_client.generate_presigned_url(
+            ClientMethod="get_object", # HTTP Method
+            Params={
+                "Bucket": bucket_name, # S3 bucket
+                "Key": object_path # relative object path
+            },
+            ExpiresIn=expiration # URL duration in seconds
+        )
+    except ClientError as err:
+        # logger.error(err) #TODO: change this
+        return None
+
+    # Return the presigned URL as a string
+    return url
+```
+Further information on this *boto3* function can be found [here](https://boto3.amazonaws.com/v1/documentation/api/latest/guide/s3-presigned-urls.html).
+
+Non-programmatically, a presigned URL can be generated using the S3 console or AWS Explorer for Visual Studio – [comprehensive documentation on presigned URLs can be found here](https://docs.aws.amazon.com/AmazonS3/latest/userguide/using-presigned-url.html). Once a presigned URL is created for a private object, it can be used within SVELT configuration files to temporarily visualize these private files.
+
+&nbsp;
+## Configuration File Creation Using Presigned URLs
+
+- all must be same cancer type
+- all must be same assembly
+
+- draw out directory structure -- include which ones contain combination of properties/files
+- give example python command with this dummy directory structure
+- list requirements.txt (i used a conda environment)
+
+- what the overall function does: generates a presigned url for every object needed in the config file. makes the config file locally, uploads to config folder within S3, then generates a presigned URL for that new timestamped config file
+- designed for: cohorts of samples with large amount of samples, with the same cancer type and assembly. good when comparing between samples within the same cohort (TODO: link out to dominika's stuff? will it be available? ask dom and sehi) 
+
+- doesnt handle notes (ask if this is needed)
+- samtools bai doesnt work on gunzipped file for me
+- check imports
+- terminal command to write to local file (> file.txt)
+- print out to terminal in the script
+- s3fs -- is this too much to mention?
+- just exceptions? or raise other type of error?
+- make public bucket for testing out?
+- tell to check properties manually in cohort view
+- generates samples and displays in order of ID list
+
+
+    #TODO: create goscan URL and put in text file (including expiration date?) and upload to S3 (maybe use sync?)
+    #TODO: look at gerstein github repo for example of walking through a script
+    # TODO: describe directory structure needed within the S3 bucket
```

Comment:
This point is important. Please explain:
- what directory structure in the bucket is expected
- what file extensions are expected
- how to link files to sample IDs